client/src/router/index.js

```diff
@@ -9,45 +9,44 @@ Vue.use(VueRouter);
 const routes = [
     {
         path: '/',
-        name: 'UserDataProvider',
         component: UserDataProvider,
         children: [
             {
                 path: '',
-                name: 'Home',
-                component: () => import('../views/Login.vue')
+                name: 'SignIn',
+                component: () => import('../views/SignIn')
             },
             {
-                path: 'about',
-                name: 'About',
-                component: () => import('../views/About.vue')
+                path: 'signup',
+                name: 'SignUp',
+                component: () => import('../views/SignUp')
+            },
+            {
+                path: 'home',
+                name: 'Home',
+                component: () => import('../views/Home')
             },
             {
                 path: 'status',
-                component: () => import('../views/Status.vue')
+                name: 'Status',
+                component: () => import('../views/Status')
+            },
+            {
+                path: 'about',
+                name: 'About',
+                component: () => import('../views/About')
             },
             {
                 path: 'profile',
-                name: 'profile',
-                component: () => import('../views/Profile.vue')
+                name: 'Profile',
+                component: () => import('../views/Profile')
             },
             {
                 path: '',
-                name: 'LoginGuard',
                 component: LoginGuard,
                 children: [
                     // There must be routes which need logged user
```

Comment:
move to LoginGuard children routes:
            {
                path: 'home',
                name: 'Home',
                component: () => import('../views/Home')
            },
            {
                path: 'status',
                name: 'Status',
                component: () => import('../views/Status')
            },
            {
                path: 'about',
                name: 'About',
                component: () => import('../views/About')
            },
            {
                path: 'profile',
                name: 'Profile',
                component: () => import('../views/Profile')
            },